### PR TITLE
add solution Homework 11.09.2019

### PR DESCRIPTION
**Pull request by Arkarine**
#3

### CODE REVIEW

File: Tasks.11.09.2019/src/test/java/FibonacciTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import static org.junit.Assert.assertEquals;
+
+public class FibonacciTest {
+    Fibonacci fibonacci;
+
+    @Before
+    public void createFibonacci(){
+        fibonacci = new Fibonacci();
+    }
+
```
Comment (sharf85) [inline Tasks.11.09.2019/src/test/java/FibonacciTest.java]: Please, add tests for edge conditions. E.g. n=0 or 1

File: Tasks.11.09.2019/src/test/java/FibonacciTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public void N_Fibonacci_Test_For10(){
+
+        assertEquals(55, fibonacci.N_Fibonacci(10));
+    }
+
+    @Test
+    public void N_Fibonacci_Test_ForBigN(){
+        assertEquals(1298777728820984005l, fibonacci.N_Fibonacci(101));
+    }
+
```
Comment (sharf85) [inline Tasks.11.09.2019/src/test/java/FibonacciTest.java]: how did you count this value for test?

File: Tasks.11.09.2019/src/test/java/AnagramTest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        assertFalse(anagram.isAnagram(s1, s2));
+    }
+
+    @Test
+    public  void isAnagramTest_CaseSensitive(){
+        String s1 = "Hello";
+        String s2 = "elOLH";
+        assertFalse(anagram.isAnagram(s1, s2));
+    }
+
```
Comment (sharf85) [inline Tasks.11.09.2019/src/test/java/AnagramTest.java]: the method should be case insensitive. But it's very minor, the problem is solved in general
